ArmoniK.Extension.CSharp.Client.DllHelper/ArmoniKServicesExt.cs

```diff
@@ -61,11 +62,13 @@ public static async Task<DllBlob> SendDllBlobAsync(this IBlobService    blobServ
                                                      SessionInfo          session,
                                                      DynamicLibrary       dynamicLibrary,
                                                      ReadOnlyMemory<byte> content,
-                                                     CancellationToken    cancellationToken)
+                                                     bool                 manualDeletion    = false,
+                                                     CancellationToken    cancellationToken = default)
```

Comment:
Can you make another PR to add missing cancellation tokens in async methids if there is any ?